src/main/java/hudson/plugins/sonar/MsBuildSQRunnerInstallation.java

```diff
@@ -95,17 +99,42 @@ public String getDisplayName() {
     }
   }
 
-  public String getExecutable(Launcher launcher) throws IOException, InterruptedException {
+  public static String getScannerName() {
```

Comment:
Avoids to duplicate the tool name in `AbstractMsBuildSQRunner.java` when the instance of this class is null.